src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -232,43 +228,26 @@ public boolean isValid() {
     }
 
     public boolean isClosedAfter(int hours) {
-        Calendar now = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
-        // Fix the time zone accordingly
-        now.add(Calendar.MILLISECOND, (int) (60 * 60 * 1000 * timeZone));
+        Date now = new Date();
 
-        Calendar start = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
-        start.setTime(startTime);
-
-        Calendar deadline = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
-        deadline.setTime(endTime);
-
-        long nowMillis = now.getTimeInMillis();
-        long deadlineMillis = deadline.getTimeInMillis();
+        long nowMillis = now.getTime();
+        long deadlineMillis = endTime.getTime();
         long differenceBetweenDeadlineAndNow = (deadlineMillis - nowMillis) / (60 * 60 * 1000);
 
-        return now.after(start) && differenceBetweenDeadlineAndNow < hours;
+        return now.after(startTime) && differenceBetweenDeadlineAndNow < hours;
```

Comment:
Not related to this PR but there's a logical flaw here. if `deadlineMillis < nowMillis` then `differenceBetweenDeadlineAndNow` will be negative and the 2nd condition will pass. Anyway, this kind of logic should be done by the `TimeHelper` class using the Java 8's time manipulation. So we'll fix this when we get to to that.

Reply:
Isn't that the correct answer?

Reply:
Oh yeah. My mistake.